File: 백준/Silver/1439. 뒤집기/뒤집기.py
c = list(input())
n0 = 0
n1 = 0

lastC = -1
for i in range(0,len(c)):
	if lastC == c[i]:
		continue
	else:
		if c[i] == '0':
			n0 += 1
		else:
			n1 += 1
	lastC = c[i]

print(n0 if n0<n1 else n1)

File: 백준/Silver/1439. 뒤집기/뒤집기.swift
import Foundation

var input = readLine()!.map{String($0)}
var cnt_0 = 0
var cnt_1 = 0

var current = ""
for s in input {
    if current == s { continue }
    else {
        current = s
        if s == "0" { cnt_0 += 1 } 
        else { cnt_1 += 1 }
    }
}
print([cnt_0, cnt_1].min()!)